Source/Blazorise/Styles/_table.scss

```diff
@@ -27,3 +27,68 @@ thead {
         position: relative;
     }
 }
+
+
+/*-- ==============================================================
+ Screen smaller than 760px and iPads.
+ ============================================================== */
+
+@media only screen and (max-width: 760px), (min-device-width: 768px) and (max-device-width: 1024px) {
```

Comment:
We might want to define media breakpoints per CSS provider since all of them have a different mobile size.